extensions/amp-video/1.0/amp-video.js

```diff
@@ -21,6 +21,29 @@ const TAG = 'amp-video';
 
 class AmpVideo extends VideoBaseElement {}
 
+/** @override */
+AmpVideo['props'] = {
```

Comment:
Moved since props vary by component, so I think they should be specified individually.